### ISSUE TITLE
Verbose and time between retries

### ISSUE THREAD

**Issue by oeresundsgruppen** [gradle/test-retry-gradle-plugin#181]
Hi, retrying seems to work, but would like to see retry number 1, 2, 3 in logs.... Also can you add a wait-time between retries, possibly going up between each ?

**Comment by marcphilipp** [comment]
In which scenarios would a wait time help?

**Comment by aluminais** [comment]
I agree with oeresundsgruppen.  Either exponential backoff or time between retries would help for my case.  Which is an external resource that can be slow to become available after it fails.  Hammering it too quickly can make things worse.
